<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pre_game_widget</class>
 <widget class="QWidget" name="pre_game_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Setup</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="player_input_widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="input_label">
        <property name="text">
         <string>Player name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_line_edit"/>
      </item>
      <item>
       <widget class="QPushButton" name="add_push_button">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="player_overview_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="overview_list_widget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="player_edit_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="remove_push_button">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="move_up_push_button">
        <property name="text">
         <string>Move up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="move_down_push_button">
        <property name="text">
         <string>Move down</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="start_game_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="start_game_push_button">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
